brush/src/components/Content.js

```diff
@@ -4,13 +4,21 @@ import Map from './map'
 import InputROMModal from './InputROMModal'
 
 const Content = () => {
-  const { romBuffer } = useContext(ROMContext)
+  const { romBufferStatus } = useContext(ROMContext)
 
-  if (romBuffer) {
-    return <Map />
-  }
+  switch (romBufferStatus) {
+    case 'starting':
+      return null
+
+    case 'empty':
+      return <InputROMModal />
 
-  return <InputROMModal />
+    case 'loaded':
+      return <Map />
+
+    default:
+      throw new Error(`Unexpected rom buffer status: ${romBufferStatus}`)
```

Comment:
If you will throw some erros like this in the app you should use some [`error boundary`](https://reactjs.org/docs/error-boundaries.html) to provide friendlier errors to the user.